index/field/field_iterator.go

```diff
@@ -58,12 +59,12 @@ type DoubleFieldIterator interface {
 	Value() float64
 }
 
-// StringFieldIterator iterates over (doc ID, string value) pairs in a string field.
-type StringFieldIterator interface {
+// BytesFieldIterator iterates over (doc ID, string value) pairs in a string field.
+type BytesFieldIterator interface {
 	BaseFieldIterator
 
 	// Value returns the current string value.
-	Value() string
+	Value() iterator.Bytes
```

Comment:
The `Bytes` type should probably not live in `iterator` package because it's generic.